libctru/include/3ds/services/nwmext.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+enum
+{
+	NWMEXT_WIFI_ENABLE = 0,
+	NWMEXT_WIFI_DISABLE = 1
+};
+
+// Initializes NWMEXT.
+Result nwmExtInit(void);
+
+// Exits NWMEXT.
+void nwmExtExit(void);
+
+/**
+ * @brief Turns wireless on or off.
+ * @param command Command to turn wireless on or off.
+ */
+Result NWMEXT_ControlWirelessEnabled(u32 command);
```

Comment:
As far as I can tell, just about all of the other services use Results, so why should this be any different? When I tested it a while back it was able to return error codes, not just a true or false boolean value.